modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -25,17 +26,22 @@ The subnets assigned to nodes and the IP addresses assigned to individual pods a
 
 While the OVN-Kubernetes network provider implements many of the capabilities present in the OpenShift SDN network provider, the configuration is not the same.
 
+* If your cluster uses any of the following OpenShift SDN capabilities, the migration to OVN-Kubernetes migrates the configuration for these features as well:
++
+--
+* Egress IPs
+* Egress firewall
+* Multicast
+--
+
 * If your cluster uses any of the following OpenShift SDN capabilities, you must manually configure the same capability in OVN-Kubernetes:
 +
 --
 * Namespace isolation
-* Egress IP addresses
-* Egress network policies
 * Egress router pods
-* Multicast
 --
 
-* If your cluster uses any part of the `100.64.0.0/16` IP address range, you cannot migrate to OVN-Kubernetes because it uses this IP address range internally.
+* If your cluster uses any part of the `100.64.0.0/16` IP address range, because OVN-Kubernetes uses this IP address range internally, you must specify an alternative IP address range for OVN-Kubernetes during the migration to avoid a conflict.
```

Comment:
I found this a little difficult to parse. Maybe it would be better as:
"If your cluster uses any part of the `100.64.0.0/16` IP address range, you must specify an alternative IP address range for OVN-Kubernetes during the migration to avoid a conflict because OVN-Kubernetes uses this IP address range internally.